### DISCUSSION TITLE
Unable to run VERACore example

### DISCUSSION THREAD

**Discussion by ylchu0424** [Kitware/trame#403] [Q&A] [answered]
Dear trame team,

The online example of VERACore (https://pvw.kitware.com/vera/) is fantastic. I've tried to download the VERACore source code (https://github.com/Kitware/VERACore) and run it locally. But it requires an VERA output file which seems missing in the git repo. I've tried to construct a .h5 file myself but it doesn't satisfy the very complicated data dimension. 

Is it possible to provide a sample file similar to the online example? 

Thanks,
Jason

**Comment by jourdain** [comment, marked as answer]
You should ask for a data file directly on VERACore and more specifically to the group of folks involved in that [PR](https://github.com/Kitware/VERACore/pull/3)
